.github/workflows/tests.yaml

```diff
@@ -69,6 +69,12 @@ jobs:
           - python-version: 3.9
             os: ubuntu-20.04
             toxenv: py39_cython
+          - python-version: 3.10.0-beta.1
```

Comment:
it seems that we can use ranges here: `"3.10.0-beta - 3.10"` automatically choses the most recent version.

I'm using it in another workflow and it works ok

Reply:
Thanks for the tip, I'll take a look at it. This is low prio though, since a bunch of packages (or our use thereof) are still broken on 3.10, namely `websockets`.